src/endToEnd.ts

```diff
@@ -3,7 +3,7 @@ import { VotesDict } from '@votingworks/ballot-encoder'
 import { E2EAPI } from './config/types'
 import fetchJSON from './utils/fetchJSON'
 
-export default async function encryptAndGetTracker(
+export default async function getBallotTrackingCode(
```

Comment:
this new name hides the fact that the ballot is being encrypted and recorded by the EG backend. Specifically, you don't want to call this function twice on the same ballot. So I would suggest a different name than something that sounds like a getter.

Reply:
Hmm… I was trying to hide this. The front end only needs a ballot tracking code and doesn't need to know about what is happening under the hood, no?